ITEvents/Services/UserSettingsService.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import CoreData
+import UIKit
+
+class UserSettingsService: IUserSettingsService {
+    let entityName = "Settings"
+    
+    func fetchSettings(then completion: @escaping (UserSettings) -> Void) {
```

Comment:
we need to add here synthetic delay for completion to imitate probably API request
And same for ```func save(settings: UserSettings)```